types/block.go

```diff
@@ -90,11 +92,11 @@ func (b *Block) ValidateBasic() error {
 	// NOTE: b.Data.Txs may be nil, but b.Data.Hash()
 	// still works fine
 	if err := ValidateHash(b.DataHash); err != nil {
-		return fmt.Errorf("Wrong Header.DataHash: %v", err)
+		return fmt.Errorf("wrong Header.DataHash: %v", err)
 	}
 	if !bytes.Equal(b.DataHash, b.Data.Hash()) {
 		return fmt.Errorf(
-			"Wrong Header.DataHash. Expected %v, got %v",
```

Comment:
Can you separate formatting changes--like changing the capitalization on this error message, which I am all in favor for--from the changes to the commit logic? It would make this PR much easier to review.

Reply:
these changes are present in a breaking linting PR, i think we can merge that after we update it, then these would go away